protected/humhub/libs/StdClass.php

```diff
@@ -0,0 +1,412 @@
+<?php
+
+/*
+ * @link      https://www.humhub.org/
+ * @copyright Copyright (c) 2023 HumHub GmbH & Co. KG
+ * @license   https://www.humhub.com/licences
+ */
+
+namespace humhub\libs;
+
+use ArrayAccess;
+use ArrayIterator;
+use Countable;
+use humhub\exceptions\InvalidArgumentValueException;
+use humhub\exceptions\InvalidArgumentTypeException;
+use OutOfBoundsException;
+use SeekableIterator;
+use Serializable;
+use Stringable;
+use Traversable;
+use yii\helpers\ArrayHelper;
+
+class StdClass extends \stdClass implements ArrayAccess, Stringable, SeekableIterator, Countable, Serializable
```

Comment:
This class seems very powerful. What are the advantages to the regular `\stdClass`. Do we really need  these additional features as well? Are they used?